Satuju/ReusableView/SwitchButton.swift

```diff
@@ -8,6 +8,8 @@
 import SwiftUI
 
 struct SwitchButton: View {
+    var action: () -> Void
+    
```

Comment:
Xcode -> Preferences -> Text Editing -> Editing -> Automatically trim trailing whitespace. And Including whitespace-only lines.